deploy/lib/server_config.rb

```diff
@@ -2685,8 +2685,9 @@ def ServerConfig.properties(prop_file_location = @@path)
     properties = ServerConfig.load_properties(default_properties_file, "ml.")
     properties.merge!(ServerConfig.load_properties(properties_file, "ml."))
 
+    #Look for optional shared_config, if it is set grab the properties from path relative to the root of the roxy project
     if properties['ml.shared_config'] then
-      shared_properties_file = ServerConfig.expand_path(properties['ml.shared_config'])
+      shared_properties_file = ServerConfig.expand_path("#{@@path}/../#{properties['ml.shared_config']}")
```

Comment:
I'm assuming we need to make the path configurable on this to fulfill your need to handle this via a submodule?